requirements/requirements.txt

```diff
@@ -478,6 +495,10 @@ slack-sdk==3.27.0
     # via -r /awx_devel/requirements/requirements.in
 smmap==5.0.1
     # via gitdb
+social-auth-app-django==5.4.2
+    # via django-ansible-base
+social-auth-core==4.5.4
+    # via social-auth-app-django
```

Comment:
@john-westcott-iv also from https://github.com/ansible/django-ansible-base/pull/633, via resource_registry, so updater script is correct.

However, I'm a little concerned about this.